includes/class-wc-stripe-webhook-handler.php

```
@@ -1603,6 +1603,48 @@ public function process_checkout_session( object $notification ): void {
+		}
+
+		$message = 'checkout.session.expired' === $notification->type ? __( 'The checkout session has expired.', 'woocommerce-gateway-stripe' ) : __( 'The async payment for this checkout session has failed.', 'woocommerce-gateway-stripe' );
+
+		$status_update = [];
+		if ( ! $order->has_status( [ OrderStatus::PROCESSING, OrderStatus::COMPLETED, OrderStatus::FAILED ] ) ) {
+			$status_update['from'] = $order->get_status();
+			$status_update['to']   = OrderStatus::FAILED;
+			$order->update_status( OrderStatus::FAILED, $message );
+		} else {
+			$order->add_order_note( $message );
+		}
```

PHPMD (2.15.0)

[error] The method process_checkout_session_failure uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)